03_ocp_repo_sync.sh

```diff
@@ -40,3 +41,12 @@ sync_go_repo_and_patch github.com/openshift/installer https://github.com/openshi
 
 # sync_go_repo_and_patch github.com/openshift/ci-operator https://github.com/openshift/ci-operator.git
 # sync_go_repo_and_patch github.com/sallyom/installer-e2e https://github.com/sallyom/installer-e2e.git
+
+sync_go_repo_and_patch github.com/metalkube/facet https://github.com/metalkube/facet.git
+
+# Build facet
+go get github.com/rakyll/statik
```

Comment:
```suggestion
go get -v github.com/rakyll/statik
```
For better visibility into what is going on.